UnionPatcher/RemotePatch.cs

```diff
@@ -121,25 +135,25 @@ public void PSNEBOOTRemotePatch(string ps3ip, string gameID, string serverURL, s
         this._ps3Mapi.PS3.Notify("UnionRemotePatcher Connected! Patching...");
 
         // Create simple directory structure
-        Directory.CreateDirectory(@"rifs");
-        Directory.CreateDirectory(@"eboot");
-        Directory.CreateDirectory($@"eboot/{gameID}");
-        Directory.CreateDirectory($@"eboot/{gameID}/original");
-        Directory.CreateDirectory($@"eboot/{gameID}/patched");
+        Directory.CreateDirectory($@"{workingDir}/rifs");
+        Directory.CreateDirectory($@"{workingDir}/eboot");
+        Directory.CreateDirectory($@"{workingDir}/eboot/{gameID}");
+        Directory.CreateDirectory($@"{workingDir}/eboot/{gameID}/original");
+        Directory.CreateDirectory($@"{workingDir}/eboot/{gameID}/patched");
 
         // Let's grab and backup our EBOOT
         FTP.DownloadFile($"ftp://{ps3ip}/dev_hdd0/game/{gameID}/USRDIR/EBOOT.BIN",
-            @$"eboot/{gameID}/original/EBOOT.BIN", user, pass);
+            @$"{workingDir}/eboot/{gameID}/original/EBOOT.BIN", user, pass);
 
         // Now we'll check and see if a backup exists on the server or not, if we don't have one on the server, then upload one
         if (!FTP.FileExists($"ftp://{ps3ip}/dev_hdd0/game/{gameID}/USRDIR/EBOOT.BIN.BAK", user, pass))
-            FTP.UploadFile(@$"eboot/{gameID}/original/EBOOT.BIN",
+            FTP.UploadFile(@$"{workingDir}/eboot/{gameID}/original/EBOOT.BIN",
                 $"ftp://{ps3ip}/dev_hdd0/game/{gameID}/USRDIR/EBOOT.BIN.BAK", user, pass);
 
         // Start getting idps and act.dat - these will help us decrypt a PSN eboot
         idps = PS3MAPI.PS3MAPIClientServer.PS3_GetIDPS();
 
-        File.WriteAllBytes(@"data/idps", IDPSHelper.StringToByteArray(idps));
+        File.WriteAllBytes($@"data/idps", IDPSHelper.StringToByteArray(idps));
```

Comment:
No, this is not as it must be the PWD from the perspective of SCETool, which is inside the app bundle.
Because it gets written every time a psn game is patched it's not like the user will be saddened by losing it. 

The only case of a problem occuring would be someone accidentally leaking their IDPS by sharing the app, so maybe it would be worth getting all this to be in a temp folder and running SCETool from there